.github/autobuild/windows.ps1

```diff
@@ -19,7 +19,7 @@ $ChocoCacheDir = 'C:\ChocoCache'
 $Qt32Version = "5.15.2"
 $Qt64Version = "5.15.2"
 $AqtinstallVersion = "2.1.0"
-$JackVersion = "1.9.17"
+$JackVersion = "1.9.21"
```

Comment:
@ann0see, the JACK version at choco has been updated. The version to install is now: 1.9.21.20220626
I have validated this on my local system and the auto install works when using that version number.

Reply:
Hmm...seems I selected the wrong option for submitting my comment.